src/candela/components/Geo/index.js

```diff
@@ -0,0 +1,33 @@
+import $ from 'jquery';
+import geojs from 'geojs/geo.js';
+import VisComponent from '../../VisComponent';
+
+export default class Geo extends VisComponent {
+
+  constructor (el, options) {
+    super(el);
+    console.log(options);
+    this.plot = geojs.map({
+                  node: el,
+                  zoom: 6,
+                  center: {x: 28.9550, y: 41.0136}
+                });
```

Comment:
This raises the larger issue of removing handlers in general. We may need a `destroy` (or `exit`) method on vis components to let them handle things like this.